### DISCUSSION TITLE
When the mouse clicks on the table component, the CPU will be 100%

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4239] [Q&A]
# try it
```
widget.NewTableWithHeaders()
```

**Comment by junmaqiang** [comment]
It seems that if you click outside the table cell, it will freeze.

**Comment by junmaqiang** [comment]
When you click on a unit, the effect will not be displayed when the mouse passes over other units.

**Comment by junmaqiang** [comment]
Sometimes I don't need the status to be displayed when the mouse passes over a table cell. Can I provide a parameter to turn off this effect?

**Comment by andydotxyz** [comment]
Please do not use discussions to report a bug. This lacks all the context we need to be able to track it down.

**Reply by junmaqiang** [reply]
im sorry , now i know

**Comment by andydotxyz** [comment]
Closing this "discussion" as it is a mix of lots of separate things which we cannot track.

1) To report an issue please open a bug report
2) To re request new functionality please open a feature request
3) To start a discussion please open with a question and keep it to that topic
